src/foam/java/InterfaceMethod.js

```diff
@@ -40,6 +40,12 @@ foam.CLASS({
       setter: function() {},
       getter: function() {}
     },
+    {
+      name: 'synchronized',
```

Comment:
This should actually be removed. Java 8 doesn't allow the `synchronized` as part of interfaces.